conftest.py

```diff
@@ -37,3 +54,59 @@ def _open_browser(url):
 
             webbrowser.open_new_tab(url)
     return _open_browser
+
+
+@pytest.fixture
+async def compare_screenshots(request):
+    """Run visual regression test on the output."""
+    async def _compare_screenshots(test_id, url):
+        if (request.config.getoption(COMPARE_SCREENSHOTS_OPTION) or
+            request.config.getoption(UPDATE_REFERENCE_SCREENSHOTS_OPTION)):
+            from playwright.async_api import async_playwright
+
+            # Filtering warnings generated by playwright
+            warnings.filterwarnings(
+                'ignore', category=DeprecationWarning, module='pyee.*')
+
+            test_dir = Path(__file__).parent / 'docrepr' / 'tests'
+            image = f'test-{test_id}-{platform}.png'
```

Comment:
```suggestion
            image = f'test-{test_id}-{sys.platform}.png'
```

To match above change